fastlane/actions/gitlab_create_project.rb

```diff
@@ -170,7 +187,10 @@ def self.migrate_issues(gitlab_src, gitlab_dst, project_src, project_dst, userma
             description: issue.description,
             assignee_id: assignee_id,
             milestone_id: milestone_id,
-            labels: issue.labels.join(",")
+            labels: issue.labels.join(","),
+            due_date: issue.due_date,
```

Comment:
it is not required. But I thought it would be nice to conserve this value as well as the other dates. 👍